ws_fnc/AI/fn_createGarrison.sqf

```diff
@@ -11,7 +12,7 @@ Minimal:
 Full:
 [center,radius,side,number,number,array,bool] call ws_fnc_createGarrison
 or
-[center,radius,side,number,number,string,bool] call ws_fnc_createGarrison
+[center,radius,side,number,number,string,bool,string] call ws_fnc_createGarrison
```

Comment:
missing zeus param here

Reply:
Fixed and fixed the other full example too